Sparkle/SUTouchBar.m

```diff
@@ -0,0 +1,94 @@
+//
+//  SUTouchBar.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBar.h"
+
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101202
```

Comment:
Does the latest SDK link the `NSTouchBar` symbol weakly? I wonder if some class-from-string trickery is needed for running on older systems.